client/CS130_Team13/Assets/Scripts/Drag n Drop/DragDropManager.cs

```diff
@@ -6,8 +6,16 @@ public class DragDropManager : MonoBehaviour
 {
     public static DragDropManager instance = null;
 
+    /// <summary>
+    /// aux object that forces the dragged item to be displayed on top
+    /// </summary>
     [SerializeField] private GameObject containerDuringDragging = null;
 
+    /// <summary>
+    /// allows other objects to have access to the dragged item
+    /// </summary>
+    public GameObject currentlyDraggedItem = null;
```

Comment:
what do you think about using getters and setters? some benefits can include 
- Encapsulation of behavior associated with getting or setting the property - this allows additional functionality (like validation) to be added more easily later.
- Insulating your public interface from change - allowing the public interface to remain constant while the implementation changes without affecting existing consumers.

for instance, we might add sounds later when we click on a item, or stuff like that